### ISSUE TITLE
Keyboard navigation on the Visio home page carousel

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#891]
## Bug Report

**Problematic behavior**
The carousel is not accessible; the user does not receive information about the content change when clicking next or previous.

**Expected behavior/code**
When navigating with the keyboard, the user should be informed that the text and decorative image have changed.

**Steps to Reproduce**
1. Log in to https://visio.numerique.gouv.fr/
2. Try to reach the carousel by tabbing
3. And try to get information about the current slide.

**Some resources:**
- [Link to the article from 24 jours de web: designing and developing a carousel accessible to people with disabilities](https://www.24joursdeweb.fr/2025/concevoir-et-developper-un-carrousel-accessible-aux-personnes-handicapees#navigation-carrousel)
- [W3C Notice](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/)

**Additional context/Screenshots**
- Be careful with the color contrast of the carousel bullets; the ratio is not sufficient between the active bullet  and the other inactive ones.
- Pay attention to the text alternatives for images that are decorative here.

<img width="518" height="687" alt="Image" src="https://github.com/user-attachments/assets/d25a3cdf-697f-40e1-8507-c25d13fd5eb2" />

**Comment by lebaudantoine** [comment]
Hi, this carousel will be removed, in a homepage refactoring

**Comment by cyberbaloo** [comment]
Hello, 
Thank you for this answer.
Do not hesitate to keep me informed when redesigning this page. :)

**Comment by cyberbaloo** [comment]
The carousel is back on the home page but also on the login page. 🫢 I'm reopening this ticket, do you need additional resources to resolve this issue?
